transforms/message-remove-classic-theme/README.md

```diff
@@ -0,0 +1,16 @@
+# message-remove-classic-theme
+
+In order to simplify the `Message` api in carbon-react
+* the `as` prop has been renamed to `variant`
+* the `roundedCorners` prop has been removed
+
+```diff
+- <Message as="info" roundedCorners>My Message</Button>
```

Comment:
The closing tag on these is `</Button>` and should be `</Message>`

Reply:
Good spot 🤦